.github/workflows/ios-app-nightly.yml

```
@@ -0,0 +1,73 @@
+name: 🍎 iOS App Nightly
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)